### PR TITLE
PIM-3575: Sort csv column in a determinist way (alphabetically) on export

### PR DESCRIPTION
**Pull request by sumbobyboys**
| Q                    | A
| -------------------- | ---
| Bug fix?             | Y
| New feature?         | N
| BC breaks?           | N
| CI currently passes? | Y
| Tests pass?          | Y 
| Scenarios pass?      | Y
| Checkstyle issues?*  | N
| PMD issues?**        | N
| Changelog updated?   | Y
| Fixed tickets        | PIM-3575
| DB schema updated?   | N
| Migration script?    | N
| Doc PR               |

### CODE REVIEW

File: features/export/convert_metric_values_during_product_export.feature [hunk 1]
```diff
@@ -47,7 +47,7 @@ Feature: Export of metric values
...
-    tshirt-black;tshirts;;men_2014;;white;;;;;;;;;;"Ein elegantes schwarz T-Shirt";"An elegant black t-shirt";"A stylish black t-shirt";"Un T-shirt noir élégant";;;;;;;american_apparel;cotton;"Schwarz T-Shirt";"Black t-shirt";"Black t-shirt";"T-shirt Black";;10.00;9.00;15.00;;size_M;;0.0000;CELSIUS;5000.0000;GRAM;1
+sku;additional_colors;categories;color;cost-EUR;cost-GBP;cost-USD;country_of_manufacture;customer_rating-ecommerce;customs_tax-de_DE-EUR;customs_tax-de_DE-GBP;customs_tax-de_DE-USD;datasheet;description-de_DE-ecommerce;description-en_GB-ecommerce;description-en_US-ecommerce;description-fr_FR-ecommerce;enabled;family;groups;handmade;image;legend-de_DE;legend-en_GB;legend-en_US;legend-fr_FR;manufacturer;material;name-de_DE;name-en_GB;name-en_US;name-fr_FR;number_in_stock-ecommerce;price-EUR;price-GBP;price-USD;release_date-ecommerce;size;thumbnail;washing_temperature;washing_temperature-unit;weight;weight-unit
```
Comment (filipsalpe) [inline features/export/convert_metric_values_during_product_export.feature]: why the change in indentation?

File: src/Pim/Bundle/BaseConnectorBundle/Writer/File/CsvWriter.php [hunk 2]
```diff
@@ -197,8 +197,13 @@ protected function getAllKeys(array $items)
         }
 
         $mergedKeys = call_user_func_array('array_merge', $intKeys);
+        $uniqueKeys = array_unique($mergedKeys);
 
-        return array_unique($mergedKeys);
+        $identifier = array_shift($uniqueKeys);
```
Comment (nidup) [inline src/Pim/Bundle/BaseConnectorBundle/Writer/File/CsvWriter.php]: "identifier" in "CsvWriter" looks a bit product oriented, as now, every entity has identifier it's maybe ok

**Comment by jjanvier** [comment]
GTM once rebased

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,7 @@
 - PIM-2103: Display a loading when deleting a category tree
 - PIM-3394: Improve forgotten password screen
 - PIM-3398: Translate units on metric fields on product edit form
+- PIM-3575: Sort csv column in a determinist way (alphabetically) on export
```
Comment (solivier) [inline CHANGELOG.md]: wrong place
